holmes.go

```diff
@@ -94,17 +105,52 @@ func (h *Holmes) EnableMemDump() *Holmes {
 	return h
 }
 
+// EnableGCHeapDump enables the GC heap dump.
+func (h *Holmes) EnableGCHeapDump() *Holmes {
```

Comment:
this name is a bit weird, maybe EnableTriggerByGCCycle is better? what's your opinion

Reply:
I think EnablexxxDump is just aligned with the existing APIs, eg EnableCPUDump.
It means to check the exception of the xxx resource, right?

yeah, the `GCHeap` may be a bit weird, it's called the HeapMarked in go source code. But I think `HeapMarked` is a bit hard to understand, so I made the `GCHeap`.
Actually, we are checking the alive heap memory size after a GC cycle.
Maybe `EnableGCHeapMarkedDump` can be better? Using an existing name may be better.

`EnableTriggerByGCCycle` is not aligning with the existing APIs and does not describe the exception of resource, but it's okay to me.